templates/components/carousel.html

```diff
@@ -63,4 +63,18 @@
     {{#and arrows (if carousel.slides.length '>' 1)}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{carousel.slides.length}}" class="js-hero-next-arrow slick-next slick-arrow"></button>
     {{/and}}
+    <button
+        type="button"
+        class="heroCarousel-play-pause-button js-hero-play-pause-button"
+        aria-label="{{lang 'carousel.playPauseButton.ariaPause'}}"
+        data-play=true
+        data-labels='{
+            "play": "{{lang 'carousel.playPauseButton.play'}}",
+            "pause": "{{lang 'carousel.playPauseButton.pause'}}",
+            "ariaPlay": "{{lang 'carousel.playPauseButton.ariaPlay'}}",
+            "ariaPause": "{{lang 'carousel.playPauseButton.ariaPause'}}"
+
+        }'>
+        {{lang 'carousel.playPauseButton.pause'}}
```

Comment:
Will we always have playing carousel as a default?

Reply:
As you can see this value is hardcoded to true
<img width="637" alt="Screenshot 2021-01-12 at 12 09 23" src="https://user-images.githubusercontent.com/66325265/104300407-1a3b7600-54cf-11eb-9526-e6a3eeb3fc8e.png">